### ISSUE TITLE
SVG icons have potential aria issues caused by unneeded id and aria attribute

### ISSUE THREAD

**Issue by zipymonkey** [psu-libraries/psulib_base#271]
The title inside the svg has an id and the svg has a aria-labeled by. These are both unnecessary and should be removed. The icon svgs are defined in the base theme so this needs to be changed there.

## Acceptance Criteria
- Remove id from all svg icon titles
- Remove aria-labeledby from svg icons

Split from https://github.com/psu-libraries/hershey/issues/548
